plugins/node/opentelemetry-instrumentation-pg/package.json

```diff
@@ -71,7 +71,7 @@
   "dependencies": {
     "@opentelemetry/core": "^1.26.0",
     "@opentelemetry/instrumentation": "^0.55.0",
-    "@opentelemetry/semantic-conventions": "1.27.0",
+    "@opentelemetry/semantic-conventions": "^1.28.0",
```

Comment:
This should be discussed.
Over in https://github.com/open-telemetry/opentelemetry-js/issues/5182 we are discussing the advice/recommendations/policy for using the "incubating" entry-point of the semconv JS package.

The rough concensus is that we will *pin* the semconv dep for a package when it is using the incubating entry-point, because in general there *can* be (hopefully rare) breaking changes in the incubating entry-point for a minor version release.

That's the reason this particular dep was pinned.

Currently the only packages in the contrib repo using the incubating endpoint are instrumentation-pg and intrumentation-aws-lambda. The latter isn't pinning the dep, mainly I assume because we are still working out how we want to handle the semconv dep.

Reply:
Pinned in [`9f2acaf` (#2549)](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2549/commits/9f2acaf8a0a5d0124e67d8d4ec81d633165d7ce2)